instance/models/mixins/openedx_config.py

```diff
@@ -245,6 +245,10 @@ def _get_configuration_variables(self):
             "DISCOVERY_HOSTNAME": '~{}'.format(self.instance.discovery_domain_nginx_regex),
             "DISCOVERY_NGINX_PORT": 80,
             "DISCOVERY_SSL_NGINX_PORT": 443,
+            # set to the openedx release being installed since the
+            # default is to install the master version which doesn't work well
+            # with other ginkgo-based services.
```

Comment:
I don't think this comment is necessary, this is true of all ``_VERSION`` variables.

Reply:
Removed in c3b1ae